stdlib/public/SDK/Foundation/PlistEncoder.swift

```diff
@@ -276,7 +283,7 @@ fileprivate struct _PlistKeyedEncodingContainer<K : CodingKey> : KeyedEncodingCo
     }
 
     mutating func nestedUnkeyedContainer(forKey key: Key) -> UnkeyedEncodingContainer {
-        let array = self.encoder.storage.pushUnkeyedContainer()
+        let array = NSMutableArray()
```

Comment:
This was the cause of a bug where requesting a nested unkeyed container inside a top-level keyed container would return an empty array for the encoded object.